src/Shared/PlatformSupport/StandardRuntimePlatform.cs

```diff
@@ -12,12 +11,16 @@ namespace Avalonia.Shared.PlatformSupport
 {
     internal partial class StandardRuntimePlatform : IRuntimePlatform
     {
+
+#if NETSTANDARD
+        public void PostThreadPoolItem(Action cb) => ThreadPool.QueueUserWorkItem(_ => cb(), null);
+#else
```

Comment:
StandardRuntimePlatform.cs is also used for iOS and Android. It's in a shared project.